webapp/src/containers/createIncident/subscribeToNewIncident.tsx

```diff
@@ -66,7 +66,7 @@ const SubscribeNewIncident = ({
                 setShowResultPanel(true);
             }
         }
-    }, [getSubscriptionState().isError, getSubscriptionState().isSuccess, getSubscriptionState().isLoading]);
+    }, [subscriptionPayload, isLoading, isError, isSuccess]);
```

Comment:
`subscriptionPayload` wasn't present here before. Is there really a need for this?

Reply:
https://github.com/mattermost/mattermost-plugin-servicenow/pull/182#discussion_r1253650334